components/inputs/input-number.js

```diff
@@ -364,16 +367,6 @@ class InputNumber extends LabelledMixin(SkeletonMixin(FormElementMixin(LocalizeC
 		});
 	}
 
-	async focus() {
```

Comment:
I didn't love that we changed the native `focus()` method to be `async`, which is why I left the `FocusMixin` implementation non-async. If folks need to be notified when it happens, they can wait for the event.

Reply:
I think @grant-cleary had to do something a little more conditional [in the editor](https://github.com/BrightspaceUI/htmleditor/blob/main/htmleditor.js#L346).  

The [underlying focus method](https://github.com/BrightspaceUI/htmleditor/blob/main/htmleditor-mixin.js) is in fact `async` in his case as well.  @grant-cleary ... you might be able to use what Dave has here... but I know your case is a little more complex.  It's not just waiting for the shadowRoot to render, it has to wait for tinyMCE and all that stuff.  Not sure if that's something you want to consider here Dave.

Reply:
Haha yeah, I was looking at this and wondering if I could make use of it. I think we're definitely still a bit hosed by having to wait for TinyMCE though.

Reply:
Yeah I did consider adding more complex options -- like a method instead of a selector or for Grant's case we could expose some kind of Promise... but at the end of the day decided against it to a) keep this simple and b) because it's pretty easy for complex cases to just implement their own `focus()`.

Reply:
Aye, what the editor does now works out well enough. Unfortunate that it needs to do such weird things, but such is life when you rely on third party packages.

Reply:
Sounds good!